### ISSUE TITLE
--download flag to jetty-start causes NullPointerException

### ISSUE THREAD

**Issue by morgwai** [eclipse/jetty.project#5844]
**Jetty version**
9.4.35

**Java version**
OpenJDK 64-Bit Server VM (build 11.0.9.1+1-Ubuntu-0ubuntu1.18.04, mixed mode, sharing)

**OS type/version**
ubuntu-18.04

**Description**

> $ java -jar ${JETTY_HOME}/start.jar --download=https://repo1.maven.org/maven2/com/h2database/h2/1.4.200/h2-1.4.200.jar\|lib/ext/h2-1.4.200.jar
java.lang.NullPointerException
	at org.eclipse.jetty.start.StartArgs.addFile(StartArgs.java:248)
	at org.eclipse.jetty.start.StartArgs.parse(StartArgs.java:1133)
	at org.eclipse.jetty.start.StartArgs.parse(StartArgs.java:1044)
	at org.eclipse.jetty.start.Main.processCommandLine(Main.java:293)
	at org.eclipse.jetty.start.Main.main(Main.java:75)

looking at [line 1133](https://github.com/eclipse/jetty.project/blob/bdc54f03a5e0a7e280fab27f55c3c75ee8da89fb/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java#L1133) it passes `null` as the first argument to `addFile`:
````
        if (arg.startsWith("--download="))
        {
            addFile(null, Props.getValue(arg));
````
which in turn immediately tries to call some method on it on [line 248](https://github.com/eclipse/jetty.project/blob/bdc54f03a5e0a7e280fab27f55c3c75ee8da89fb/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java#L248):
````
    private void addFile(Module module, String uriLocation)
    {
        if (module.isSkipFilesValidation())
````
so it would be strange if it didn't result in a NullPointerException  ;-)

**Comment by morgwai** [comment]
affects also version 10.0.0:
>java.lang.NullPointerException
	at org.eclipse.jetty.start.StartArgs.addFile(StartArgs.java:250)
	at org.eclipse.jetty.start.StartArgs.parse(StartArgs.java:1142)
	at org.eclipse.jetty.start.StartArgs.parse(StartArgs.java:1053)
	at org.eclipse.jetty.start.Main.processCommandLine(Main.java:309)
	at org.eclipse.jetty.start.Main.main(Main.java:76)

**Comment by morgwai** [comment]
how about 10.0.x branch?

**Comment by sbordet** [comment]
Work in 9.4.x is merged forward (unless specific to 9.4.x only), so it's in 10.0.x and 11.0.x too.
